collectors/diskspace.plugin/README.md

```diff
@@ -1,6 +1,11 @@
 # diskspace.plugin
 
-This plugin monitors the disk space usage of mounted disks, under Linux.
+This plugin monitors the disk space usage of mounted disks, under Linux. The plugin requires Netdata to have execute/search permissions on the mount point itself, as well as each component of the absolute path to the mount point. You can create a separate group to control access to mount points, include `netdata` user to the group, and set the execute/search permissions using
+
+```sh
+chown :volumeadm /mnt/volume
+chmod g+x /mnt/volume
+```
```

Comment:
I'm going to be entirely honest, I do not like having the example.

For most 'normal' users, it's kind of useless as the collector will just work for them since the default permissions for a mount point in most cases will include world search permissions.  And, if somebody who doesn't know better just blindly follows it for all the mount points on their system, it will break their system.

For those users who are likely to be affected by this issue, the ideal solution is too system specific to post an example (for example, if the filesystem type of the mount supports POSIX ACLs, then adding a specific ACL to give the Netdata user search permissions is almost always going to be a better solution than the given example), and this particular example is likely to cause any of a wide variety of subtle issues if it even works to begin with (and it won't on a well secured system using a MAC system like SELinux, which is the most likely setup to cause this issue in the first place).

Reply:
I agree with @Ferroin. @cakrit - your turn :).

Reply:
As if I will ever disagree with @Ferroin ... Respect.